history/titles/c_dawnstar.txt

```diff
@@ -6,3 +6,14 @@
 1.1.1={
 	holder = 42082
 }
+603.9.1={
+    effect = {
+        location = {
+			add_province_modifier = { name = depopulated_3 duration = -1 }
+		}
+	}
+}
+604.1.1={
+	holder = 42075
+	liege = k_quelthalas
```

Comment:
but already has `liege=e_quelthalas` above?

Reply:
Fixed in 5e37d69f8f88612528194e58ca91f3e2cb46686f.